### PR TITLE
[Show Feed] Don’t scroll the hero units view.

### LINKED ISSUE #953: banner should not pull down and separate from status bar
image

### PR DESCRIPTION
**Pull request by alloy**
Fixes #953.

### CODE REVIEW

**Comment by alloy** [comment]
Wow, so many reviewers! Assigning @orta.

File: Podfile.lock [hunk 1]
```diff
@@ -309,7 +309,7 @@ SPEC CHECKSUMS:
   OCMock: a6a7dc0e3997fb9f35d99f72528698ebf60d64f2
   OHHTTPStubs: c93230597bc5c8b74bcfdb403cc8773515b8b08c
   ORKeyboardReactingApplication: 05af5ead40af968d5423d54b6a0dc050858b0e82
-  ORStackView: 5df6b1b990b0648d8ef6f69f89361ea8648e8f64
+  ORStackView: 752d81ca2e2a0bd6c01a2ece30fd24923434be1e
   ReactiveCocoa: fbe689fe218063b7bbed41032912c9ca4fdbe172
   RSSwizzle: d561958f595028bfcef98c7d55c318b3878a61c6
   SDWebImage: 1d2b1a1efda1ade1b00b6f8498865f8ddedc8a84
```
Comment (orta) [inline Podfile.lock]: should we worry about these?

**Comment by orta** [comment]
Looks good to me!

**Comment by orta** [comment]
if we're not worried about the Podfile, then you can opt to add a changelog entry and that's it

**Comment by alloy** [comment]
I’m not worried about those checksums changing, as long as it doesn’t keep happening between our machines.

**Pull request merged**